data/migrations/20190112093745_project_table.js

```diff
@@ -0,0 +1,14 @@
+
+exports.up = function(knex, Promise) {
+  return knex.schema.createTable('projects', tbl => {
+      tbl.increments();
+    //   tbl.integer().notNullable().unsigned();
+      tbl.string('project_name', 255).unique();
+      tbl.string('project_description', 255);
+      tbl.boolean('project_completed').notNullable().defaultTo(false);
```

Comment:
Having `project_` seems repetitive since these are for the `projects` table. Why not just take that part out of the names?